Helper/Activity.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Metrilo\Analytics\Helper;
+
+class Activity extends \Magento\Framework\App\Helper\AbstractHelper
+{
+    public function __construct(
+        \Metrilo\Analytics\Helper\Data $dataHelper
+    ) {
+        $this->dataHelper = $dataHelper;
+    }
+    
+    public function createActivity($storeId, $type)
+    {
+        $token    = $this->dataHelper->getApiToken($storeId);
+        $secret   = $this->dataHelper->getApiSecret($storeId);
+        $endPoint = $this->dataHelper->getApiEndpoint();
```

Comment:
Isn't this still wrong? Answer these questions:

1. Where do we post tracking in QA?
2. Where do we post activity in QA?
3. Where do we post tracking in Production?
4. Where do we post activity in Production?

Reply:
1. qaurl.customerboard.co
2. qaurl.customerboard.co
3. tracking.metrilo.com
4. p.metrilo.com

Is that correct ?

Reply:
For 1 & 2 what would the urls be if the name of the env is "plg-124">

Reply:
1. shop-plg-124.customerboard.co
2. shop-plg-124.customerboard.co

Reply:
> 1. Where do we post tracking in QA?
> 2. Where do we post activity in QA?

`shop-plg-124.customerboard.co` is the shop URL

Reply:
1. tracking-plg-124.customerboard.co
2. p.metrilo.com/monolith-web-plg-124.customerboard.co

Reply:
Yep. So does the code solve this?

Reply:
Not like that so i have to add another configuration like metrilo_analytics/general/activity_endpoint witch will have values 'http://p.metrilo.com or $ACTIVITY_ENDPOINT for prod/qa ?

Reply:
yeah, sounds like a plan